### DISCUSSION TITLE
Release 2.7.4/2.7.5

### DISCUSSION THREAD

**Discussion by revoxhere** [revoxhere/duino-coin#1103] [General]
### What's new, what's changed...
* Added new languages for the AVR Miner (Italian :it:, German :de:, Azerbaijani :azerbaijan: and Dutch :netherlands:)
* Added Portuguese :portugal: locale to the PC Miner
* Removed XXHASH in the PC Miner by @revoxhere 
* Added mining intensity setting in the PC Miner by @revoxhere 
* Removed unnecessary delay in Arduino Code by @revoxhere 
* Add Emoji support for Windows Terminal by @lockness-Ko in https://github.com/revoxhere/duino-coin/pull/1033
* HTTPS /getPool support for ESP32 by @ProtoThis in https://github.com/revoxhere/duino-coin/pull/1047
* HTTPS /getPool support for ESP8266 by @ProtoThis in https://github.com/revoxhere/duino-coin/pull/1048
* Rewrote mining job for ESP32 by @ProtoThis in https://github.com/revoxhere/duino-coin/pull/1058
* Added print semaphore by @moth-boi in https://github.com/revoxhere/duino-coin/pull/1087
* Fixed Discord Presence in the AVR Miner by @moth-boi in https://github.com/revoxhere/duino-coin/pull/1091
* Added and updated some benchmarks by @primitt in https://github.com/revoxhere/duino-coin/pull/1095
* Fix a typo in the AVR Miner by @ernesto75 in https://github.com/revoxhere/duino-coin/pull/1102

**Note:** ESP8266 and 32 codes are already on version 2.7.5, this is because they received more updates over that time and it would be confusing to go back to 2.7.4 for them again. Mac users, please [launch from source](https://github.com/revoxhere/duino-coin#linux) as it causes less problems.

### New contributors - welcome!
* @lockness-Ko made their first contribution in https://github.com/revoxhere/duino-coin/pull/1033
* @ProtoThis made their first contribution in https://github.com/revoxhere/duino-coin/pull/1047
* @moth-boi made their first contribution in https://github.com/revoxhere/duino-coin/pull/1087
* @hotmoist made their first contribution in https://github.com/revoxhere/duino-coin/pull/1093
* @ernesto75 made their first contribution in https://github.com/revoxhere/duino-coin/pull/1102

**Full changelog**: https://github.com/revoxhere/duino-coin/compare/2.7.3...2.7.4

#### ⚠️ NOTICE: .exe binaries may be detected as a virus by your antivirus software. This is a **false positive** caused by pyinstallers bootloader, read about it [here](https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg). Duino-Coin is not a virus.

As always, please report any encountered issues. Happy mining!

<hr /><em>This discussion was created from the release <a href='https://github.com/revoxhere/duino-coin/releases/tag/2.7.4'>Release 2.7.4/2.7.5</a>.</em>

**Comment by Makcukin** [comment]
How do if I want to install it for my PC wich is using 32-bit system?

**Comment by kosa6661987** [comment]
Is it any chance to solve issue with picopi? Fighting a month already with that. Thankyou .

**Comment by JACK7JACK** [comment]
How code Arduino using android please help me

**Reply by JACK7JACK** [reply]
Sorry how to code
